src/auth0-sso-login.js

```diff
@@ -37,16 +36,16 @@ export default class auth {
 
   /**
    * @description Get the latest available profile
-   * @param auth0AccessToken The client specific audience token to call Auth0 Management
    * @return {null|Object} profile if the user was already logged in; null otherwise
    */
-  getProfile(auth0AccessToken) {
+  getProfile() {
     return Promise.resolve()
     .then(() => {
       let idToken = this.getIdToken();
       let jwt = jwtManager.decode(idToken);
+      let auth0AccessToken = this.authResult && this.authResult.idToken;
```

Comment:
Might be better to name this idToken ;)

Reply:
This IS the toket to accessAuth0, I already call the thing about `idToken`, to keep consistency, I'm leaving this alone, sorry.